Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/Inputs/TextInput.swift

```diff
@@ -179,6 +196,50 @@ private extension TextInput {
     }
 }
 
+@available(iOS 16.0, *)
+private extension TextInput {
+    /// A view for displaying a multiline text input outside the body of the feature form view.
+    ///
+    /// By moving outside of the feature form view's scroll view we let the text field naturally manage
+    /// keeping the input caret visible.
+    struct FullScreenTextInput: View {
+        /// The current text value.
+        @Binding var text: String
+        
+        /// An action that dismisses the current presentation.
+        @Environment(\.dismiss) private var dismiss
+        
+        /// A Boolean value indicating whether the text field is focused.
+        @FocusState private var textFieldIsFocused: Bool
+        
+        /// The element the input belongs to.
+        let element: FieldFormElement
+        
+        var body: some View {
+            InputHeader(element: element)
+            TextField(
+                element.label,
+                text: $text,
+                prompt: Text(element.hint).foregroundColor(.secondary),
+                axis: .vertical
+            )
+            .focused($textFieldIsFocused, equals: true)
+            .onAppear {
+                textFieldIsFocused = true
+            }
+            Spacer()
+            HStack {
+                InputFooter(element: element)
+                Button("Done") {
```

Comment:
The [HIG](https://developer.apple.com/design/human-interface-guidelines/sheets) states:

_Position Done and Cancel buttons as people expect. Typically, a Done or Dismiss button belongs in a sheet’s top-right corner (in a left-to-right layout) or top-left corner (in a right-to-left layout). The Cancel button belongs in a sheet’s top-left (in a left-to-right layout) or top-right (in a right-to-left layout) corner._

So putting the "Done" button in the upper right would be preferable.